### ISSUE TITLE
Morphia 2.0 Compatibility ?!

### ISSUE THREAD

**Issue by javajack** [ganchix/morphia-spring-boot-starter#30]
The maintainer of morphia has released 2.0 with rejuvenated activity on this project.

https://morphia.dev/

Is there any plan to keep this starter maintained for 2.0 onwards compatibility ?!

**Comment by ganchix** [comment]
Yes, of course, I'm working on it you can see the progress in the following feature branch: https://github.com/ganchix/morphia-spring-boot-starter/tree/feature/morphia2.0

Support the new version of morphia changes the minimum version of java that the starter support, now it will be 11.
Also, I'm going to update Spring Boot version to support the latest version.

Thanks for your comments.
